CONTRIBUTORS.md

```diff
@@ -20,52 +20,6 @@ Contributors of Apache MXNet (incubating)
 MXNet has been developed by a community of people who are interested in large-scale machine learning and deep learning.
 Everyone is more than welcomed to is a great way to make the project better and more accessible to more users.
 
-Committers
```

Comment:
List is outdated and doesn't seem necessary to be maintained.

Reply:
I see,I made a mistake when changing it using notepad++.I will fix it

Reply:
@wkcn I just worked for another thing and didn't find that the if-else statement wasn't added in the new version.Thanks for your correction.